src/NemoStuff.jl

```diff
@@ -46,10 +46,6 @@ base_ring(::Vector{Int}) = Int
 #
 ################################################################################
 
-function zero_matrix(::Type{MatElem}, R::Ring, n::Int)
-    return zero_matrix(R, n)
-end
-
```

Comment:
why do you delete this?

Reply:
I think @lgoettgens wanted to delete this before and I objected with some argument, which I don't want to rehash again.

Reply:
Because JET complained because there is no method for `zero_matrix` taking a ring and a single integer... which is correct, at least for AA...

Of course we could add such a thing if people think this is useful, but maybe not?

Reply:
I would find this useful. `identity_matrix` needs a single Int as well, so why not just add it